cmd/handlers.c

```diff
@@ -197,3 +245,114 @@ cmdAddCallback(CmdPacket *cmd) {
 
   cmdAddCb(name, cmd->value); // save the sensor callback
 }
+
+// Query the number of wifi access points
+static void ICACHE_FLASH_ATTR cmdWifiGetApCount(CmdPacket *cmd) {
+  int n = wifiGetApCount();
+  os_printf("WifiGetApCount : %d\n", n);
```

Comment:
The `os_printf` in this file need to be converted to `DBG` so they can be disabled. See 8 lines up, for example.